Rakefile

```diff
@@ -66,7 +66,14 @@ namespace :db do
     url = EventSourceryTodoApp.config.database_url
     database_name = File.basename(url)
     database = Sequel.connect URI.join(url, '/template1').to_s
-    database.run("CREATE DATABASE #{database_name}")
+    database.run(<<~DB_QUERY)
+      DROP DATABASE IF EXISTS #{database_name};
```

Comment:
This helps make the script a little more idempotent, for example, failed migration or some other problem. It's just easier to drop existing databases and start from scratch IMO.

Reply:
I don't think I prefer having this in shared repositories. Maybe if you find you are doing this a lot you can add an alias in your `.zshrc`?

Reply:
Thanks @spaxcell  

This was a bit of an oversight on my end. It might be a little surprising running `rake db:create` and see your database getting dropped.

I've reverted this in [b228989](https://github.com/envato/event_sourcery_todo_app/pull/30/commits/b2289894b38cabd73142fe1c8ef80f23165035ba)